lib/arclight/solr_ead_indexer_ext.rb

```diff
@@ -126,5 +130,31 @@ def add_collection_creator_to_component(node, solr_doc)
       creators = node.xpath('//archdesc/did/origination[@label="creator"]/*/text()').map(&:text)
       solr_doc[field_name] = creators - [repository]
     end
+
+    def parent_check_list(node, root_path, element_path, results = [])
```

Comment:
This method is similar to the `ancestral_visit` method above and might be reuseable.